rj_gameplay/rj_gameplay/skill/temp_pivot_skill.py

```diff
@@ -0,0 +1,53 @@
+from abc import ABC, abstractmethod
+
+import rj_gameplay.eval as eval
+import argparse
+import py_trees
+import sys
+import time
+import numpy as np
+from typing import Optional
+
+import stp.skill as skill
+import stp.role as role
+import stp.action as action
+from rj_gameplay.action import pivot
+from stp.skill.action_behavior import ActionBehavior
+import stp.rc as rc
+from rj_msgs import msg
+
+class IPivot(skill.ISkill, ABC):
+    ...
+
+
+"""
+A skill version of pivot so that actions don't have to be called in tactics
```

Comment:
This is a common enough occurrence that I wonder if we should just make a "action skill" that directly feeds through.

Reply:
^ Agreed. I think most of our skills have this comment.

If I'm reading [this](https://github.com/UBC-Thunderbots/Software/blob/master/docs/software-architecture-and-design.md#strategy) correctly, UBC's stack simply doesn't have skills. Instead, their actions do things like:

> 1. Moving to a position (without colliding with anything)
> 2. Shooting the ball at a target
> 3. Intercepting a moving ball

which sounds like a direct motion planner = skill correspondence to me.

Reply:
This specific skill is a temp one for testing, but yeah for other actions which require direct skill wrappers, this makes sense.